js/RecentDonations.js

```diff
@@ -0,0 +1,196 @@
+(function() {
+var app = angular.module('RecentDonations', []);
+app.controller('RecentDonationsControl', function($scope) {
+
+    var urlVars = GLOBALS.getUrlVars();
+    var count = Number(urlVars['count']);
+    var uid = urlVars['id'];
+    var isTeam = urlVars['team'];
+
+    if (count === undefined) { count = 5 }
+
+    var contributions = []
```

Comment:
_good practice_ requires semicolons at the end of every line. Alex is rebelling for some reason.

But yeah Alex it really is safer if you put semicolons....